google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/GoogleApiClientAppEngineAdminService.java

```diff
@@ -159,7 +161,7 @@ private Operation getOperation(
       @NotNull String projectId, @NotNull String operationName, @NotNull Credential credential)
       throws IOException {
     // The operation ID is the final slash-separated component of the operation name.
-    String[] nameParts = operationName.split("/");
+    String[] nameParts = Iterables.toArray(Splitter.on("/").split(operationName), String.class);
```

Comment:
was trying to make the least invasive changes here to quiet the warning. if you think just adding `@SuppressWarnings("StringSplitter")` is better let me know.

Reply:
Maybe use `splitToList`? Kind of unusual to see String.class. It looks like List would work too